app/tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Houston tests
+
+This file contains some technical information for test files.
+
+### Overview
+
+All tests are expected to be ran in parallel and return a promise of an object in the form of:
+```javaScript
+{
+  error: ['error message strings'],
+  warning: ['warning message strings']
+}
+```
+
+NOTE: If you expect to save data to the database, use the `.update()` function instead of `.save()`. This will prevent data overwrites when running tests in parallel.
```

Comment:
I'm not sure I understand this — when would a test want/need to save data to the database?